app/src/main/res/layout/activity_main.xml

```diff
@@ -2,17 +2,272 @@
 <androidx.constraintlayout.widget.ConstraintLayout xmlns:android="http://schemas.android.com/apk/res/android"
     xmlns:app="http://schemas.android.com/apk/res-auto"
     xmlns:tools="http://schemas.android.com/tools"
+    android:id="@+id/Top_part"
     android:layout_width="match_parent"
     android:layout_height="match_parent"
-    tools:context=".MainActivity">
+    android:background="@color/grey_background"
+    android:padding="@dimen/main_padding">
 
     <TextView
+        android:id="@+id/maine_title_logo"
         android:layout_width="wrap_content"
         android:layout_height="wrap_content"
-        android:text="Hello World!"
-        app:layout_constraintBottom_toBottomOf="parent"
-        app:layout_constraintLeft_toLeftOf="parent"
-        app:layout_constraintRight_toRightOf="parent"
+        android:text="@string/big_logo"
+        android:textColor="@color/black"
+        android:textSize="@dimen/Extra_big_header_text_size"
+        android:textStyle="bold"
+        app:layout_constraintBottom_toTopOf="@+id/steps_containner"
+        app:layout_constraintEnd_toEndOf="parent"
+        app:layout_constraintStart_toStartOf="parent"
         app:layout_constraintTop_toTopOf="parent" />
 
-</androidx.constraintlayout.widget.ConstraintLayout>
\ No newline at end of file
+    <TextView
+        android:id="@+id/second_title"
+        android:layout_width="wrap_content"
+        android:layout_height="wrap_content"
+        android:layout_marginBottom="@dimen/main_text_marginBottom"
+        android:text="@string/connected_to"
+        android:textColor="@color/black"
+        android:textSize="@dimen/Small_header_text_size"
+        app:layout_constraintBottom_toTopOf="@+id/three_dots_menu"
+        app:layout_constraintEnd_toEndOf="parent"
+        app:layout_constraintStart_toStartOf="parent"
+        app:layout_constraintTop_toBottomOf="@+id/maine_title_logo" />
+
+    <ImageView
+        android:id="@+id/three_dots_menu"
+        android:layout_width="wrap_content"
+        android:layout_height="@dimen/three_dots_menu_height"
+        android:layout_margin="@dimen/three_dots_menu_margin"
+        android:src="@drawable/three_dots"
```

Comment:
try background

Reply:
background applied

Margin should be saved because 3 dots does not take the right place without margin